src/test/java/Homework21.java

```diff
@@ -0,0 +1,31 @@
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class Homework21 extends BaseTest {
+    @Test
+    public void renamePlaylist()  {
+        //setUpBrowser();
+        //String url = "https://bbb.testpro.io/";
+        //driver.get(url);
+        //getBrowser();
+        emailField("skillztomkid@gmail.com");
+        passwordFinder("te$t$tudent");
+        logIn();
+        editPlaylist();
+        notificationIcon();
+        Assert.assertTrue(notificationVerification());
+
+        //editPlaylist();
+        //searchSongs("dark days");
+        //viewAll();
+        //clickFirstSong();
+       // addToPlaylist();
+       // newPlaylist("Ope");
+       // opePlaylist();
+       // deletePlaylist();
+       // notificationIcon();
+       // Assert.assertTrue(notificationVerification());
+       // driver.wait(5000);
```

Comment:
**I recommend not using code comments:** for, example, when you have a code that cannot be understood, then comments are not the solution. Try to rewrite the code or rename fields and other elements in a way that the reader will understand the action that you are doing there.